components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -3757,16 +3764,16 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
 
         await this.guardWithFeatureFlag("orgGitAuthProviders", user, newProvider.organizationId);
 
+        // Ensure user can perform this operation on this organization
```

Comment:
nit: not sure such a comment is helpful, because (hopefully) the single call below the comment expressed that well enough.

Reply:
Cargo cult, can remove :+1: